code/modules/research/rdconsole.dm

```diff
@@ -658,7 +658,6 @@ won't update every console in existence) but it's more of a hassle to do. Also,
 		if(1.0) //Main Menu
 			dat += "<div class='statusDisplay'>"
 			dat += "<h3>Main Menu:</h3><BR>"
-			dat += "<A href='?src=\ref[src];menu=1.1'>Current Research Levels</A><BR>"

```

Comment:
Why are you removing this.

Reply:
That removes the link to the page which I have merged into the main menu, as for the title screen, how do I remove it?